build/all_projects.settings.targets

```diff
@@ -20,6 +20,9 @@
     <DropDir>$(MIEngineRoot)bin\$(Configuration)</DropDir>
     <ToolsHome>$(MIEngineRoot)\tools</ToolsHome>
     <IntermediateOutputPath>$(MIEngineRoot)obj\$(Configuration)\$(MSBuildProjectName)\</IntermediateOutputPath>
+
+    <CanGenerateMDBFiles>false</CanGenerateMDBFiles>
+    <CanGenerateMDBFiles Condition="$(Configuration.Contains('Desktop')) AND $([MSBuild]::IsOSPlatform('windows'))">true</CanGenerateMDBFiles>
```

Comment:
>CanGenerateMDBFiles [](http://example.com/codeflow?start=122&length=19)

Did you figure out if we actually still need these, or if we can just use portable PDBs?

Reply:
I was going to keep this since we technically are still running as mono?

Reply:
Right, but mono has supported portable PDBs for a long time now, so I am not sure if you need them. Probably worth trying to debug and see if the mono will just happily use the portable ones.

Reply:
So.... MonoDevelop is now only `Build From Source Only`.
![image](https://user-images.githubusercontent.com/3953714/124333612-968f4780-db49-11eb-8e24-d09a7ccaabe5.png)

Putting it on halt for .net core plan.